faithcomesbyhearing/dbp
# FCBH-681 Backend Error Reporting sentry.io

# Description
- Installed and configured Sentry.io PHP SDK to report errors

## Issue Link
Original Story: [FCBH-681](https://fullstacklabs.atlassian.net/browse/FCBH-681) 
Review Story: [FCBH-821](https://fullstacklabs.atlassian.net/browse/FCBH-821) 

## How Do I QA This
- Run `composer install`
- Add to the `.env` file the `SENTRY_DSN=` key. (This key is on the Jira Story)
- On the `.env` file change `APP_ENV=local` to `APP_ENV=production`
- Open `http://dbp.test/wiki/bibles/undefined` to trigger an error
- Verify on `Sentry.io` that the error was reported

## Checklist
